CORE/world_gadgets/configuration_engine.rb

```diff
@@ -20,7 +20,9 @@ def merge_system_config
     @config_data.keys.each do |option_name|
       if ENV[option_name]
         puts "Loading value [#{ENV[option_name]}] for ENV variable [#{option_name}] from system environment."
-        @config_data[option_name] = ENV[option_name]
+        bool_map = { "true"=>true, "false"=>false }
+        option_value = bool_map.keys.include?(ENV[option_name]) ? bool_map[ENV[option_name]] : ENV[option_name]
```

Comment:
`bool_map.keys.include?(ENV[option_name]) ? ...` could just be `bool_map[ENV[option_name]] ? ...`

Reply:
Another option:
```
option_value = option_value=="true" ? true : option_value
option_value = option_value=="false" ? false : option_value
```

Reply:
i like the second option so far best.

Reply:
Me too :)

Reply:
i would rather add a .to_bool to String